guides/hosting/configurations/opentelemetry.md

```diff
@@ -0,0 +1,109 @@
+---
+nav:
+  title: OpenTelemetry
+  position: 20
+
+---
+
+# OpenTelemetry
+
+OpenTelemetry is a standard to collect distributed traces, metrics and logs from the application. It's similar to tools like NewRelic, DataDog, Blackfire Monitoring and Tideways, but it's completely open source and vendor neutral. That means you can use OpenTelemetry to collect the data and push it to one of the vendors mentioned above, or you can use it to collect the data and push it to your own infrastructure with tools like Grafana Stack (Tempoo, Loki, Prometheus, Grafana) or other tools.
+
+## Requirements
+
+To use OpenTelemetry with Shopware, you need to have the following requirements met:
+
+- `ext-opentelemetry` [PHP extension](https://github.com/open-telemetry/opentelemetry-php-instrumentation)
+- `ext-grpc` (optional, required when the transport method is gRPC)
+
+## Installation
+
+To install the OpenTelemetry Shopware extension, you need to run the following command:
+
+```bash
+composer require shopware/opentelemetry
+```
+
+This will install the OpenTelemetry Shopware bundle and create new configuration file `config/packages/prod/opentelemetry.yaml` with Symfony Flex plugin.
+
+This file configures in production that the Shopware Profiler integration with OpenTelemetry is enabled and also configures that the monolog output will be sent to OpenTelemetry.
+
+## Configuration
+
+After the installation, you will need to set some environment variables to configure the OpenTelemetry itself and exporter.
+
+### Basic configuration
+
+Enables the OpenTelemetry auto-instrumentation and sets the service name.
+
+```
+OTEL_PHP_AUTOLOAD_ENABLED=true
+OTEL_SERVICE_NAME=shopware
+```
+
+### Exporter configuration
+
+The OpenTelemetry extension needs to be configured to export the data to your collector. Here is an example todo that with with the OTLP exporter with GRPC:
+
+```
+OTEL_TRACES_EXPORTER=otlp
```

Comment:
do i have other config options? seems weird as the option is already prefixed with OTEL :thinking: 

IMHO it should either be:
```
OTEL_TRACES_EXPORTER=1
```
or
```
TRACES_EXPORTER=otlp
TRACES_EXPORTER=blackfire
```

Reply:
These are OTEL specific environment variables, we can't rename them 😅 
https://opentelemetry.io/docs/instrumentation/php/sdk/#configuration

Reply:
gotcha, then link it and describe that those envs come from the sdk and are not shopware specific